lua/gitblame/init.lua

```diff
@@ -115,6 +122,9 @@ local function load_blames(callback)
             on_stdout = function(data)
                 process_blame_output(blames, filepath, data)
                 if callback then callback() end
+            end,
+            on_exit = function()
+                files_data_loading[filepath] = false
```

Comment:
Please set it to `nil` instead in order to remove it from the table:

```lua
files_data_loading[filepath] = nil
```

Reply:
Fixed!